charts/kueue/values.yaml

```diff
@@ -0,0 +1,86 @@
+# Default values for kueue.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+nameOverride: ""
+fullnameOverride: ""
+
+# Enable each function, like kustomize https://github.com/kubernetes-sigs/kueue/blob/main/config/default/kustomization.yaml
+enablePrometheus: false
+enableCertManager: false
```

Comment:
our default is true https://github.com/kubernetes-sigs/kueue/blob/main/apis/config/v1beta1/defaults.go#L71

Reply:
Thanks. Fixed it.